src/SqlAsyncCollector.cs

```diff
@@ -521,6 +529,7 @@ public static async Task<TableInformation> RetrieveTableInformationAsync(SqlConn
                         }
                         caseSensitiveSw.Stop();
                         TelemetryInstance.TrackDuration(TelemetryEventName.GetCaseSensitivity, caseSensitiveSw.ElapsedMilliseconds, sqlConnProps);
+                        logger.LogDebugWithThreadId($"END GetCaseSensitivity Duration={caseSensitiveSw.ElapsedMilliseconds}ms Query=\"{getDatabaseCollationQuery}\"");
```

Comment:
You should log the queries at the start - since if they fail they'll never hit these lines